app/controllers/api/v1/users_controller.rb

```diff
@@ -60,7 +60,9 @@ def update
   end
 
   def index
-    if logins = params.delete(:login).try(:split, ',').try(:map, &:downcase)
+    if api_user.is_admin? && emails = params.delete(:email).try(:split, ',').try(:map, &:downcase)
```

Comment:
perhaps refactor the params extraction, split, downcase map to a reusable method here?